MPChartLib/src/main/java/com/github/mikephil/charting/renderer/LineChartRenderer.java

```diff
@@ -570,8 +570,13 @@ public void drawValues(Canvas c) {
                     }
 
                     if (entry.getIcon() != null && dataSet.isDrawIconsEnabled()) {
```

Comment:
If entry.getIcon() throws, it will already fail here. Can you post a stacktrace of the crash?

Reply:
https://github.com/Freeyourgadget/Gadgetbridge/issues/960#issuecomment-361715727 The first one.

Reply:
IOW, the line number with the NPE should have been 572, not 574.

Reply:
This would also mean that 5e5bd6c1d3e95c515d4853647ae554e48ee1d593 should work around this crash.

Reply:
If that commit works around then it would be nice if you'd update GB to use that.

Reply:
I think this PR can be closed until something that weird stacktrace is seen again.